tests/test_fetch_firmware.py

```
@@ -0,0 +1,594 @@
+        mock_url_response.status_code = 200
+        mock_url_response.text = 'https://example.com/firmware.zip'
+        mock_url_response.raise_for_status = Mock()
+
+        mock_ver_response = Mock()
+        mock_ver_response.status_code = 200
+        mock_ver_response.text = 'Version_1.0'
+        mock_ver_response.raise_for_status = Mock()
+
+        mock_get.side_effect = [mock_url_response, mock_ver_response]
+
+        result = get_from_oos_api('15', 'EU')
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)